src/app/containers/Include/amp/Idt2Amp.jsx

```diff
@@ -11,7 +11,7 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
 const Idt2Amp = ({ imageBlock }) =>
   imageBlock ? (
     <IncludeGrid>
-      <AmpImg fallback={false} {...imageBlock} />
+      <AmpImg {...imageBlock} />
```

Comment:
From the [docs](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/placeholders/?format=websites) it looks like the `fallback` attribute can be applied to any HTML element as long as it's a direct child of the AMP element